docker-compose.yaml

```diff
@@ -42,14 +42,30 @@ services:
     profiles:
       - local
 
+  redis:
+    image: redis:8.0-M03-alpine
+    command: redis-server --save 60 1 --loglevel warning
+    profiles:
+      - local
+    volumes:
+      - redis-data:/data
```

Comment:
Should we add a `make` target to allow the user to clean this, if they choose to?

Reply:
We have talked about using labels to prune the right containers. IIRC the volumes can get pruned too with a certain flag. I'll check open PRs to see how far we've done of this. IMO this volume should have the same life cycle as the ray and redis containers, so there should be no need to clean it up separately.